packages/composite-checkout/src/components/checkout-order-summary.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { useLineItems } from '../public-api';
+
+export default function CheckoutOrderSummary() {
+	const [ items ] = useLineItems();
+
+	return (
+		<React.Fragment>
+			<ProductList>
+				{ items.map( ( product, index ) => {
+					return <ProductListItem key={ index }>{ product.label }</ProductListItem>;
```

Comment:
It's not a good idea in general to use `index` as a key. Here's a quote from [the React docs on keys](https://reactjs.org/docs/lists-and-keys.html#keys):

> We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state.

In this case, each element of the `items` array will have a unique `id` attribute as guaranteed by our schema for items, so we can use `product.id` as the key.

Reply:
Good to know. Thanks for the suggestion.